packages/maskbook/src/plugins/ITO/SNSAdaptor/ClaimAllDialog.tsx

```diff
@@ -322,55 +332,69 @@ export function ClaimAllDialog(props: ClaimAllDialogProps) {
         hasOnlyOneChild: true,
     }
     return (
-        <InjectedDialog open={open} onClose={onClose} title={t('plugin_ito_claim_all_dialog_title')}>
-            <DialogContent className={classes.wrapper}>
-                <AbstractTab {...tabProps} />
-                <div className={classes.contentWrapper}>
-                    {loading || loadingOld || initLoading || !swappedTokens || !swappedTokensOld ? (
-                        <div className={classes.emptyContentWrapper}>
-                            <CircularProgress size={24} />
-                        </div>
-                    ) : swappedTokens.length > 0 || swappedTokensOld.length > 0 ? (
-                        <>
-                            {swappedTokensOld.length > 0 ? (
-                                <div className={classes.content}>
-                                    {swappedTokens.length > 0 && swappedTokensOld.length > 0 ? (
-                                        <Typography color="textPrimary" className={classes.contentTitle}>
-                                            {t('plugin_ito_claim_all_old_contract')}
+        <SnackbarProvider
+            domRoot={DialogRef.current as HTMLElement}
```

Comment:
https://iamhosseindhv.com/notistack/demos#react-portal

Otherwise, snackbar would be covered under shadow root dialog